src/testdir/test_autocmd.vim

```diff
@@ -4871,11 +4871,11 @@ func Test_GuiEnter_Turkish_locale()
     let lng = v:lang
     lang tr_TR.UTF-8
     let result = execute(':au GuiEnter')
-    call assert_equal("\n--- Autocommands ---", result)
+    call assert_match("\n--- \\(Autocommands\\|Otokomutlar\\) ---", result)
     let result = execute(':au GUIENTER')
-    call assert_equal("\n--- Autocommands ---", result)
+    call assert_match("\n--- \\(Autocommands\\|Otokomutlar\\) ---", result)
     let result = execute(':au guienter')
-    call assert_equal("\n--- Autocommands ---", result)
+    call assert_match("\n--- \\(Autocommands\\|Otokomutlar\\) ---", result)
```

Comment:
Maybe `gettext("\n--- Autocommands ---")` can be used for this?

Reply:
Ah, I will try it.